cmd/swarm/main.go

```diff
@@ -177,8 +177,11 @@ func init() {
 		CorsStringFlag,
 		EnsAPIFlag,
 		SwarmTomlConfigPathFlag,
+		//swap flags
```

Comment:
if we indicate where the swap flags begin, we should be clear where they end (or: where the next set of flags begin)